tests/child_thread/README.md

```diff
@@ -0,0 +1,4 @@
+This directory tests enclave with child thread:
+1. Do ecall in child thread.
+2. Destroy enclave in child thread
```

Comment:
@jinghe-INTC It would be good to elaborate what behavior we are trying to lock down by each of these tests.
For example, via *2*, are we just trying to check if we can destroy an enclave from a thread other than the main thread or is there more to this test?